modules/send_email.nf

```diff
@@ -0,0 +1,15 @@
+// sends an e-mail to the submitter(s)
```

Comment:
This may be out of scope for this ticket, but we should be thinking of ways to make steps like this modular so that we can send different types of emails (post-validation, post-scoring, post-evaluation, etc.). Look at `score.nf` and `validate.nf` for examples.

Reply:
Per this [comment](https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/pull/4#discussion_r1475238679) thread, different types of emails in the sense I describe above are likely not necessary. We can add the ability to send different emails (for different challenges perhaps) in the future if need be.